benchexec/tablegenerator/__init__.py

```diff
@@ -1382,10 +1386,201 @@ def write_head_line(
         out.write("\n")
 
 
+def write_tex_command_table(
+    out,
+    run_sets: List[RunSetResult],
+    stats: List[List[ColumnStatistics]],
+    **kwargs,
+):
+    bench_name_set = set()
+    for benchmark in run_sets:
+        bench_name_formatted = LatexCommand.format_command_part(
+            benchmark.attributes.get("benchmarkname")
+        )
+        if bench_name_formatted in bench_name_set:
+            logging.error(
+                "Duplicated formatted benchmark name %s detected. Benchmark names must be unique for Latex"
```

Comment:
Hm, I think this would be a common situation. As we can see it does already complicate our tests if this is not supported.

First, the benchmark name alone does not need to be unique, does it? Only the combination of benchmark and run set need to be unique AFAIU.
Second, can't we simply add a suffix with a number if the names are identical?